example/android/app/build.gradle

```diff
@@ -144,12 +144,12 @@ android {
         }
     }
     signingConfigs {
-        release {
-            storeFile file(MEMBRANE_UPLOAD_STORE_FILE)
-            storePassword MEMBRANE_UPLOAD_STORE_PASSWORD
-            keyAlias MEMBRANE_UPLOAD_KEY_ALIAS
-            keyPassword MEMBRANE_UPLOAD_KEY_PASSWORD
-        }
+//        release {
```

Comment:
You commited this by accident.
I'll fix that now so that you don't need to comment that out.